packages/veritone-react-common/src/components/Breadcrumbs/BreadcrumbItem.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
```

Comment:
This doesn't need to be in its own file either.... there is no special logic here. Just put it in `index.js`

Reply:
In my opinion, BreadcrumbItem should be in its own file to make it more redable

Reply:
👍